### PR TITLE
ISSUE-3 Fix vehicle model

### PR DESCRIPTION
**Pull request by saksofon997**
Done

### CODE REVIEW

File: MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html [hunk 1]
```diff
@@ -100,6 +100,19 @@
...
+      <div class="col-sm-8 d-flex justify-content-sm-start justify-content-center mt-2">
+        <mat-form-field class="example-full-width">
+          <input formControlName="availableMileage" matInput type="number">
+          <mat-error *ngIf="!form.get('availableMileage').valid || form.get('availableMileage').touched">
+            Available mileage is required
+          </mat-error>
+        </mat-form-field>
+      </div>
+      <div class="w-200"></div> -->
+
```
Comment (mihajlo-perendija) [inline MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html:115]: Whaat

File: MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html [hunk 2]
```diff
@@ -183,17 +205,17 @@
...
           <div class="" *ngIf="!form.get('unlimitedMileage').value" [@inOutAnimation]>
             <mat-form-field class="example-full-width" style="width: 90%;">
-              <input formControlName="alowedMileage" matInput type="number">
-              <!-- <mat-error *ngIf="!form.get('mileage').valid || form.get('mileage').touched">
-              Mileage is required
-            </mat-error> -->
+              <input formControlName="availableMileage" matInput type="number">
+              <mat-error *ngIf="!form.get('availableMileage').valid || form.get('availableMileage').touched">
+              Available mileage is required
+            </mat-error>
```
Comment (mihajlo-perendija) [inline MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html:218]: Whaat

File: MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.ts [hunk 3]
```diff
@@ -159,6 +175,14 @@ export class VehicleAdministrationComponent implements OnInit {
         alert(error);
       }
     );
+    this.pricelistService.getByOwner(1).subscribe( //TODO USER ID
```
Comment (mihajlo-perendija) [inline MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.ts:178]: User service works

File: MainApp/Servers/search/src/main/java/search/repository/SearchRepo.java [hunk 4]
```diff
@@ -21,6 +21,8 @@
             "or ((occ.startTime >= (:start_time)) and (occ.startTime < (:end_time)))" +
             "or ((occ.endTime > (:start_time)) and (occ.endTime <= (:end_time)))))" +
 
+            //"and ((:available_mileage is null) or (:available_mileage) <= (vehicle.availableMileage))" +
+
```
Comment (mihajlo-perendija) [inline MainApp/Servers/search/src/main/java/search/repository/SearchRepo.java:25]: Wut

File: MainApp/Servers/search/src/main/resources/data-postgres.sql [hunk 5]
```diff
@@ -113,44 +113,44 @@ INSERT INTO search.pricelist(
...
+    id, brand_id, model_id, category_id, transmission_id, fuel_id, seats, child_seats, mileage, cdw, pricelist_id, number_of_stars, number_of_reviews, location_latitude, location_longitude, deleted, owner_id, available_mileage)
+    VALUES (nextval('vehicle_id_seq'), 3, 8, 1, 4, 2, 5, 1, 190000, true, 5, 4, 1, 45.2553823, 19.8317609, false, 1, -1); /*7*/
 INSERT INTO search.vehicle( -- Mazda MX-5 8
-    id, brand_id, model_id, category_id, transmission_id, fuel_id, seats, child_seats, mileage, cdw, pricelist_id, number_of_stars, number_of_reviews, location_latitude, location_longitude, deleted, owner_id)
-    VALUES (nextval('vehicle_id_seq'), 4, 9, 3, 1, 1, 2, 0, -1, false, 3, 5, 1, 45.2553823, 19.8317609, false, 2); /*8*/
+    id, brand_id, model_id, category_id, transmission_id, fuel_id, seats, child_seats, mileage, cdw, pricelist_id, number_of_stars, number_of_reviews, location_latitude, location_longitude, deleted, owner_id, available_mileage)
+    VALUES (nextval('vehicle_id_seq'), 4, 9, 3, 1, 1, 2, 0, 200000, false, 3, 5, 1, 45.2553823, 19.8317609, false, 2, 15000); /*8*/
```
Comment (mihajlo-perendija) [inline MainApp/Servers/search/src/main/resources/data-postgres.sql:138]: Who is gonna ride 15000 miles in one go?  I'd go around 300~1500

File: MainApp/Servers/vehicle/src/main/java/vehicle/service/impl/VehicleServiceImpl.java [hunk 6]
```diff
@@ -78,6 +78,7 @@ public Vehicle convertToModel(VehicleDTO vehicleDTO) throws ConversionFailedErro
 
         List<VehicleDTO> vehicleDTOS = new ArrayList<VehicleDTO>();
         for (Vehicle vehicle: vehicles) {
+            System.out.println(vehicle.getAvailableMileage());
```
Comment (mihajlo-perendija) [inline MainApp/Servers/vehicle/src/main/java/vehicle/service/impl/VehicleServiceImpl.java:81]: Hehe

File: MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html (hunk 1 above)
Comment (saksofon997) [reply MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html:115]: Whoops, forgot to delete

File: MainApp/Servers/search/src/main/java/search/repository/SearchRepo.java (hunk 4 above)
Comment (saksofon997) [reply MainApp/Servers/search/src/main/java/search/repository/SearchRepo.java:25]: Added

File: MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html (hunk 2 above)
Comment (coperope) [reply MainApp/Client/src/app/vehicle-page/vehicle-administration/vehicle-administration.component.html:205]: Sta ti nije jasno?

**Pull request merged**